cypress/support/commands.ts

```diff
@@ -71,6 +81,17 @@ function deleteUserByEmail(email: string) {
 Cypress.Commands.add("login", login);
 Cypress.Commands.add("cleanupUser", cleanupUser);
 
+// We're waiting a second because of this issue happen randomly
+// https://github.com/cypress-io/cypress/issues/7306
+// Also added custom types to avoid getting detached
+// https://github.com/cypress-io/cypress/issues/7306#issuecomment-1152752612
+// ===========================================================
+function visitAndCheck(url: string) {
+  cy.visit(url);
+  cy.location("pathname").should("contain", url).wait(1000);
```

Comment:
Is 1000 milliseconds necessary? Seems long 🤔

Reply:
Il have a play around and try find the shortest time we can use. Or could also just have a sending parameter for wait time.

Reply:
I wonder whether there's a way to turn this into something generic: https://github.com/cypress-io/cypress/issues/7306#issuecomment-1154997447 🤔

Reply:
I did try the `if (Cypress.dom.isDetached($el)) cy.wait(500);` But couldn't get it to work constantly which was annoying.

I shorted the wait to `100` but also made it someone could add in their own if they need longer.